.travis.yml

```diff
@@ -2,3 +2,9 @@ language: dart
 dart: dev
 script: ./tool/travis.sh
 sudo: false
+branches:
+  only:
+    - master
```

Comment:
Um, actually, that's exactly what I thought it meant.  I realize that for the workflow of "push a branch, then immediately make a pull request" it's duplicate work.  But that's not the only workflow.  Another useful workflow is "push a branch, then look at the travis status to determine whether additional work is needed before making a push request."  I think this workflow is useful too, and if the duplicate work in the former case isn't costing too much, I think it's worth putting up with it in order to allow the latter case.

How much is the duplicate work costing us, anyhow?